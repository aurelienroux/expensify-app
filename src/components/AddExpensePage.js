import React from 'react';

const AddExpensePage = () => (
  <div>
    add expenses!
  </div>
);

export default AddExpensePage;