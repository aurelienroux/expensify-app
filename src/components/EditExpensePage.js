import React from 'react';

const EditExpensePage = (props) => (
  <div>
    edit expenses number {props.match.params.id}
  </div>
);

export default EditExpensePage;