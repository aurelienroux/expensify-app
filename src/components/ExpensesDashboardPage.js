import React from 'react';
import ExpenseList from './ExpenseList';

const ExpensesDashboardPage = () => (
  <div>
    <ExpenseList />
  </div>
);

export default ExpensesDashboardPage;