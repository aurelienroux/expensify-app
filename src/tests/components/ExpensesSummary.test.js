import React from 'react'
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render ExpensesSummary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={123} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={2} expensesTotal={12377} />)
  expect(wrapper).toMatchSnapshot();
});
