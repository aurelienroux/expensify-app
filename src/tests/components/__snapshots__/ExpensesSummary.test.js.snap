// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $1.23
  </h1>
</div>
`;

exports[`should render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling 
    $123.77
  </h1>
</div>
`;
